### ISSUE TITLE
Update color palette

### ISSUE THREAD

**Issue by ggeisler** [projectblacklight/arclight#521]
We currently have many occurrences of insufficient color contrast, mainly because Bootstrap's primary blue is doesn't have sufficient contrast on white or light gray, and white on the primary blue (as in the primary button styling) also fails.

So while we do not want to have an opinionated color palette for the base app, we should adjust the colors we use enough that the base app doesn't have color contrast problems.

Updated color palette TBD.

**Comment by seanaery** [comment]
Resolving this issue as part of a review of `arclight 1.0.x`. These specific contrast issues are now fixed.
